### DISCUSSION TITLE
When can I upgrade to v3 version?

### DISCUSSION THREAD

**Discussion by good-ideal** [wailsapp/wails#2557] [General]
### Describe the solution you'd like

When can I upgrade to v3 version?

**Comment by KiddoV** [comment]
We are all excited, but I guess we just have to wait 😉

**Reply by good-ideal** [reply]
It's been too long since I last paid attention to this repository. How's the progress of V3? Is there any hope of releasing V3 this year?

**Reply by leaanthony** [reply]
Beta soon! :-)
